usaspending_api/database_scripts/matviews/functions_and_enums.sql

```diff
@@ -5,10 +5,10 @@ CREATE EXTENSION IF NOT EXISTS intarray;
 
 -- The function(s) and enum(s) below were originally created for the materialized views
 
-CREATE TYPE total_obligation_bins AS ENUM ('<1M', '1M..25M', '25M..100M', '100M..500M', '>500M');
+CREATE TYPE public.total_obligation_bins AS ENUM ('<1M', '1M..25M', '25M..100M', '100M..500M', '>500M');
```

Comment:
Is this line change necessary? If yes, I would say we want the functions to match this explicitness too to be consistent

Reply:
Answer is, it depends. Depends on what the `search_path` is for the session executing the query. In a normal psql session, it will stay as default, which would create the object in `public` schema, unless a schema for the user's name exists.

I think though to be explicit, we should add it to all, so it will work regardless of a person's set `search_path` and  any schemas they have on their machine. Will make the change.